File: 프로그래머스/주식가격.java
import java.util.*;
class Solution {
    public int[] solution(int[] prices) {
        List<Integer> answer = new ArrayList<Integer>();
        
        
        for(int i = 0 ; i < prices.length; i++){
            boolean flag = false;
            
            for(int j = i+1 ; j < prices.length; j++ ){
                if (prices[j] < prices[i]){
                    answer.add(j-i);
                    flag = true;
                    break;
                }
            }
            if(flag == false ){
                answer.add(prices.length - i - 1);
            }
        }
        
        int[] answer2 = answer.stream().mapToInt(i->i).toArray();
        
        return answer2;

    }
}

File: 프로그래머스/주식가격.py
def solution(prices):
    
    answer = []
    
    for i in range(len(prices)):
        flag = False
        for j in range(i+1, len(prices)):
            if prices[j] < prices[i]:
                answer.append(j-i)
                flag = True
                break
        if flag == False:
            answer.append(len(prices) - i - 1)
        
    return answer